src/manipulation.js

```diff
@@ -202,7 +202,10 @@ function domManip( collection, args, callback, ignored ) {
 								jQuery._evalUrl( node.src );
 							}
 						} else {
-							DOMEval( node.textContent.replace( rcleanScript, "" ), doc );
+							if ( node.hasAttribute && node.hasAttribute( "nonce" ) ) {
+								nonce = node.getAttribute( "nonce" );
+							}
+							DOMEval( node.textContent.replace( rcleanScript, "" ), doc, nonce );
```

Comment:
As @timmywil mentioned, this needs a rebase, after which it should be as simple as adding "nonce" to [`preservedScriptAttributes`](https://github.com/jquery/jquery/blob/625e19cd9be4898939a7c40dbeb2b17e40df9d54/src/core/DOMEval.js#L6)—no changes to the manipulation module required.

Reply:
@gibson042 Adding `nonce` to `preservedScriptAttributes` should work and would be the preferred way of solving this issue. Do you want me to provide a separate PR since this one seems to be quite old and inactive? Are there any plans when jquery 3.4 will be released? I think this issue should be of high priority.

Reply:
Sure, that would be welcome.